Source/UZKArchive.m

```diff
@@ -387,7 +387,7 @@ - (BOOL)extractFilesTo:(NSString *)destinationDirectory
         return NO;
     }
     
-    NSFileManager *fm = [NSFileManager defaultManager];
+    NSFileManager *fm = [[NSFileManager alloc] init];
```

Comment:
Yes, I think I read somewhere that it was best to create your own NSFileManager if you were doing anything in the background.